assets/stylesheets/mobile/chat-drawer.scss

```diff
@@ -0,0 +1,107 @@
+.full-page-chat {
+  .channels-list {
+    min-height: 100vh;
+    padding-bottom: 6rem;
+
+    .chat-channel-row {
+      padding: 0.5rem 1rem;
+      margin: 0;
+
+      &:not(:last-child) {
+        &::after {
+          content: "";
+          display: block;
+          width: 90%;
+          padding-top: 20px;
+          border-bottom: 1px solid var(--primary-low);
+          margin: auto;
+          position: absolute;
+          bottom: 0;
+          left: 0;
+          right: 0;
+        }
+      }
+    }
+
+    .chat-channel-divider {
+      padding-inline: 2rem;
+      font-size: var(--font-up-1);
+
+      .channel-title {
+        color: var(--quaternary);
+      }
+    }
+
+    .channels-list-container {
+      background: var(--primary-very-low);
+      border-radius: 0.5rem;
+      margin: 1rem;
+    }
+
+    .chat-user-avatar {
+      img {
+        width: calc(var(--chat-mobile-avatar-size) - 2px);
+        height: calc(var(--chat-mobile-avatar-size) - 2px);
+      }
+
+      + .chat-channel-title__usernames {
+        margin-left: 1rem;
+      }
+    }
+
+    .chat-channel-title {
+      &__users-count {
+        width: var(--chat-mobile-avatar-size);
+        height: var(--chat-mobile-avatar-size);
+        padding: 0;
+        font-size: var(--font-up-3);
+        justify-content: center;
+
+        & + .chat-channel-title__name {
+          margin-left: 1rem;
+        }
+      }
+
+      &__name {
+        font-size: 20px;
+        font-size: var(--font-up-2);
+      }
+
+      &__category-badge {
+        font-size: var(--font-up-2);
+      }
+    }
+
+    .chat-channel-unread-indicator {
+      margin-left: 0;
+    }
+  }
+
+  .btn-floating.new-dm {
+    position: absolute;
+    background: var(--tertiary);
+    bottom: 2.5rem;
+    right: 2.5rem;
+    border-radius: 50%;
+    font-size: var(--font-up-5);
+    padding: 1rem;
+    transition: transform 0.25s ease, box-shadow 0.25s ease;
```

Comment:
what does this do? dont see any effect and not sure how the effect would be triggered anyway?

Reply:
@chapoi This affects the blue floating action button. It's a subtle micro-interaction so that when you press down on the floating button with your finger it slightly scales down along with the shadow changing so it mimics something being pressed down.

Reply:
Ah okay now I see it, cool!